packages/shared/src/mock/metadata.ts

```diff
@@ -56,6 +56,14 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
       withImageAttributes(makeDataset('threeD_rgb', intType, [3, 3, 3])),
       makeDataset('fourD', intType, [3, 9, 20, 41]),
     ]),
+    makeGroup('typed_arrays', [
+      makeDataset('uint8', intType, [2, 2]),
+      makeDataset('int16', intType, [2, 2]),
+      makeDataset('float32', intType, [2, 2]),
+      makeDataset('float64', intType, [2, 2]),
+      withImageAttributes(makeDataset('uint8_rgb', intType, [2, 2, 3])),
+      withImageAttributes(makeDataset('float32_rgb', intType, [2, 2, 3])),
+    ]),
```

Comment:
Adding some mock datasets. Since all values except `float32` are converted to `float32`, I haven't bothered covering every possible typed array.

Reply:
`*_rgb` datasets should have the shape `[3, 3, 3]` to display all the values in `rgb`. 

Otherwise, we end up only with the first 4 values that only includes channels that are `0` or `255` which makes us miss problems for other values.

Reply:
Oh right, well spotted :+1: